pihole-google.txt

```diff
@@ -2297,7 +2300,6 @@ fonts.googleapis.com
 4u74fojwgn.com
 5.chart.apis.google.com
 527.drive.google.com
-53.26.241.35.bc.googleusercontent.com
```

Comment:
Please try not to delete lines and add them somewhere else, it's almost impossible to review the additions like this because it looks like you have deleted them.

Reply:
good afternoon: I apologize for that, but I imagine that this would happen to me when I transfer the connections that I am detecting and others. And after adding them, I check for duplicates. I'll try to be more careful. Sorry for the inconveniences.

Reply:
Thank you! You can probably solve this to first add them to the bottom of the file, then check for duplicates, then place the new domains in de right category.